### PR TITLE
Updated sbt-spiewak

### PR DESCRIPTION
**Pull request by djspiewak**
Nothing was really broken here, it just had to be updated to the new APIs.

### CODE REVIEW

**Comment by johnynek** [comment]
@djspiewak thanks so much for taking a look at this, I really appreciate it.

related to #154

File: build.sbt [hunk 1]
```diff
@@ -62,9 +62,10 @@ ThisBuild / githubWorkflowAddedJobs ++= Seq(
 ThisBuild / githubWorkflowPublish ++= Seq(
   WorkflowStep.Sbt(List("docs/makeSite")),
   WorkflowStep.Use(
-    "JamesIves",
-    "github-pages-deploy-action",
-    "3.7.1",
+    UseRef.Public(
+      "JamesIves",
+      "github-pages-deploy-action",
+      "3.7.1"),
```
Comment (regadas) [inline build.sbt:68]:
```suggestion
    UseRef.Public("JamesIves", "github-pages-deploy-action", "3.7.1"),
```
`scalafmt` complaining about this. @djspiewak hope you don't mind going ahead with this.

Comment (regadas) [reply build.sbt]: 🤔 so I thought I would not burden you with a simple format but we have another pickle. 

Due to this change https://github.com/djspiewak/sbt-spiewak/pull/42 `MiMa` previous artifact `cats-parse` `0.3.0` is now used correctly.

```
java.lang.IllegalArgumentException: MiMa supports Scala 2.11-2.13, not 3.0.0-M3
```

`MiMa` `0.8.1` effectively doesn't have Scala `3` support. Support was added https://github.com/lightbend/mima/pull/587 but not yet released.

**Comment by larsrh** [comment]
This PR is haunted, so I merged #170 instead.

**Comment by johnynek** [comment]
Thanks all!

**Pull request closed without merging**